docs/source/cli_help/traverse.rst

```diff
@@ -65,6 +65,23 @@ It will pick up the walk from the current branch (unless ``--start-from=`` or ``
 Unlike with e.g. ``git rebase``, there is no special ``--continue`` flag, as ``traverse`` is stateless
 (doesn't keep a state of its own like ``git rebase`` does in ``.git/rebase-apply/``).
 
+The rebase and push behaviour of ``traverse`` can also be customized for each branch separately using branch qualifiers.
+There are ``rebase=no`` and ``push=no`` qualifiers that can be used to opt out of default behaviour (rebasing and pushing).
```

Comment:
Also... TBD if it makes sense to mention the qualifiers in the help for `status` and/or `format` 🤔 I would say it doesn't harm (esp. that `status` is aware of qualifiers, i.e. renders them in a special manner)